src/test/java/org/opensearch/geospatial/action/upload/geojson/UploadGeoJSONRequestTests.java

```diff
@@ -36,17 +40,22 @@ private String getRequestBody(Map<String, Object> contentMap) {
 
     public void testStreams() throws IOException {
         String requestBody = getRequestBody(null);
-        UploadGeoJSONRequest request = new UploadGeoJSONRequest(new BytesArray(requestBody.getBytes(StandardCharsets.UTF_8)));
+        RestRequest.Method method = PUT;
```

Comment:
nit: why we use variable to keep request method in this test case, but use constant direct in other test case? I think we the same approach is used in both methods is makes code look more consistent.

Reply:
This test case checks request input parameters and i am testing getMethod() by asserting down whether method is same as actual or not. In other test case, it was merely an input parameter.